src/components/media/stories/Media.stories.tsx

```diff
@@ -18,7 +18,7 @@ export const example = () => (
         </Media.Figure>
 
         <Media.Body>
-            <Media.Menu label="Options">
+            <Media.Menu>
```

Comment:
`label` prop is not accepted for `Button`

Reply:
I think that should be `aria-label` then - it's meant to provide accessible text for the DropdownMenu label